config/webpack.config.manual-resolves.js

```diff
@@ -39,7 +39,6 @@ module.exports = {
         'cozy-realtime',
         'cozy-stack-client',
         'cozy-ui',
-        'date-fns',
```

Comment:
What does it do? We still have multiple date-fns versions, right?

Reply:
Oui justement c'est ce qu'on veut, ce qui n'était pas possible avant